addons/hr_holidays/models/hr_department.py

```diff
@@ -1,30 +1,38 @@
 # -*- coding: utf-8 -*-
-import datetime
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
```

Comment:
this file already in new API. so no need to change any stuff. so can you revert changes of this file ?